### PR TITLE
Code Cleanup in various classes and XML files

### LINKED ISSUE #142: Code Cleanup in various classes and XML files
Post code-review, the following changes need to be made:
1.  In the manifest.xml file:  the code supporting version 4.0 and earlier which defines parent activity can be deleted as our minimum api is 21 (Android 5.0)
2.  In MainActivity.java, the method onStickyClicked(String string) is unnecessary.  Move this code into the onTopicItemClick() method.
3. In MainActivity.java:  In onCreateOptionsMenu(Menu menu), instead of calling return Super.onCreateOptionsMenu(menu); simplify by calling return true;
4.  In ResourceListActivity.java:  Remove log statements as they are expensive memory-wise in the final APK.
5.  In ResourceListActivity.java:  call isConnected() from Utilities class and alert user with Toast that they need to enable a connection instead of the generic toast message in the catch block
6.  In ResourceListAdapter.java:  In the onBindViewHolder() method, there is a setTag call to add the resourceId to the viewholder's tag property which is never retrieved.  Remove this line.
7.  In ResourceListAdapter.java:  In the onBindViewHolder() method, the resourceFormat if/else block would be cleaner and more readable as a switch case block.

### PR DESCRIPTION
**Pull request by JaeW**
Removed meta data parent activity tags in manifest which support versions 4.0 and older
Removed onStickyClicked(String string) method from MainActivity - redundant
Removed log statements from ResourceListActivity
Added check for active connection in ResourceListActivity upon user click to launch internet resource
Remove setTag() call from onBindViewHolder() in ResourceListAdapter
Change if/else block to switch statement in ResourceListAdapter

Fixes #142 
- 
-

### CODE REVIEW

File: app/src/main/java/com/tiptopgoodstudio/androidresources/db/entity/Resource.java [hunk 1]
```diff
@@ -23,8 +23,7 @@
     private String resourceUrl;
     private String resourceFormat;
 
-    // Adding an empty constructor in order to work with Firebase importing data into this POJO
-    // By Divya on 4/24/2018
+    // empty constructor requireed by Firebase to importi data into this POJO
```
Comment (dambadipudi) [inline app/src/main/java/com/tiptopgoodstudio/androidresources/db/entity/Resource.java]: Small typos in the comments, thats all.

File: app/src/main/java/com/tiptopgoodstudio/androidresources/ui/ResourceListActivity.java [hunk 2]
```diff
@@ -15,211 +15,142 @@
...
-
     }
 
     /**
-     * This method retrieves data from the ViewModel
-     *
-     * Added by Divya on 4/20/2018.
-     *
+     * Retrieves LiveData List<Topic> from ViewModel and sets this
+     * activity as LiveDatta Observer - updates adapter data on LiveData change
```
Comment (dambadipudi) [inline app/src/main/java/com/tiptopgoodstudio/androidresources/ui/ResourceListActivity.java]: Small typo in the comment.

File: app/src/main/java/com/tiptopgoodstudio/androidresources/ui/adapters/ResourceListAdapter.java [hunk 3]
```diff
@@ -51,33 +41,34 @@ public ResourceListAdapterViewHolder onCreateViewHolder(ViewGroup parent, int vi
...
-        }
-        else {
-            if(resourceFormat.equalsIgnoreCase("pdf")) {
-                resourceId = R.drawable.resource_pdf_icon;
-            } else if(resourceFormat.equalsIgnoreCase("video")) {
-                resourceId = R.drawable.resource_video_icon;
-            } else if(resourceFormat.equalsIgnoreCase("image")) {
-                resourceId = R.drawable.resource_image_icon;
+        if (resourceFormat == null) {
+            resourceFormat = RESOURCE_TYPE_PDF;
```
Comment (dambadipudi) [inline app/src/main/java/com/tiptopgoodstudio/androidresources/ui/adapters/ResourceListAdapter.java]: The default format needs to "url"

File: app/src/main/java/com/tiptopgoodstudio/androidresources/ui/adapters/ResourceListAdapter.java [hunk 4]
```diff
@@ -29,11 +13,17 @@
 
 import java.util.List;
 
+/**
+ * Adapter for ResourceListActivity RecyclerView
+ */
 public class ResourceListAdapter
-        extends RecyclerView.Adapter<ResourceListAdapter.ResourceListAdapterViewHolder>{
+        extends RecyclerView.Adapter<ResourceListAdapter.ResourceListAdapterViewHolder> {
+
```
Comment (dambadipudi) [inline app/src/main/java/com/tiptopgoodstudio/androidresources/ui/adapters/ResourceListAdapter.java]: There needs to be another String RESOURCE_TYPE_URL = "url"

File: app/src/main/java/com/tiptopgoodstudio/androidresources/ui/adapters/ResourceListAdapter.java [hunk 5]
```diff
@@ -51,33 +41,34 @@ public ResourceListAdapterViewHolder onCreateViewHolder(ViewGroup parent, int vi
...
+                    resourceId = R.drawable.resource_pdf_icon;
+                    break;
+                case RESOURCE_TYPE_VIDEO:
+                    resourceId = R.drawable.resource_video_icon;
+                    break;
+                case RESOURCE_TYPE_IMAGE:
+                    resourceId = R.drawable.resource_image_icon;
+                    break;
+                default:
+                    resourceId = R.drawable.resource_pdf_icon;
```
Comment (dambadipudi) [inline app/src/main/java/com/tiptopgoodstudio/androidresources/ui/adapters/ResourceListAdapter.java]: Even in the switch statement, the default needs to be R.drawable.resource_url_icon

**Comment by JaeW** [comment]
Changes made per @dambadipudi code review.

**Pull request merged**